app/views/content/landing/mailing-list-2/_header.html.erb

```diff
@@ -1,6 +1,5 @@
-<%= render Content::CampaignHeroComponent.new(
-  title: ["Get personalised", "guidance"],
+<%= render Content::LandingHeroComponent.new(
```

Comment:
When there is no image, the title abuts the left-hand side on mobile - is this intended? Should there be a bit of padding?

Will leave you to your expert view @jenhadfield-dfe - I don't have a strong opinion!

![image](https://github.com/DFE-Digital/get-into-teaching-app/assets/87642394/ffdcc317-336b-45fa-a8f8-3d82b602caf9)